doc/en/user/source/styling/workshop/ysld/polygon.rst

```diff
@@ -0,0 +1,1024 @@
+   
+   .. note:: Technically the boundary of a polygon is a specific case of a LineString where the first and last vertex are the same, forming a closed LinearRing.
+
+#. The effect of adding **stroke** is shown in the map preview:
+   
+   .. image:: ../style/img/polygon_fill_2.png
+
+#. An interesting technique when styling polygons in conjunction with background information is to control the fill opacity.
+
+   The **fill-opacity** property is used to adjust transparency (provided as range from 0.0 to 1.0). Use of **fill-opacity** to render polygons works well in conjunction with a raster base map. This approach allows details of the base map to shown through.
+
+   The **stroke-opacity** property is used in a similar fashion, as a range from 0.0 to 1.0.
+
+   .. code-block:: yaml
+      :emphasize-lines: 4,7
+      
+      symbolizers:
+      - polygon:
+          fill-color: 'white'
+          fill-opacity: 0.5
+          stroke-color: 'lightgrey'
+          stroke-width: 0.25
+          stroke-opacity: 0.5
+
+#. As shown in the map preview:
+
+   .. image:: ../style/img/polygon_fill_3.png
+   
+#. This effect can be better appreciated using a layer group.
+   
+   .. image:: ../style/img/polygon_fill_4.png
+   
+   Where the transparent polygons is used lighten the landscape provided by the base map.
+
+   .. image:: ../style/img/polygon_fill_5.png
+   
+.. only:: instructor
+     
+   .. admonition:: Instructor Notes 
+    
+      In this example we want to ensure readers know the key property for polygon data.
+    
+      It is also our first example of using opacity.
+
+Pattern
+-------
+
+The **fill-graphic** property can be used to provide a pattern. 
+
+.. image:: ../style/img/PolygonPattern.svg
+
+The fill pattern is defined by repeating one of the built-in symbols, or making use of an external image.
+
+#. We have two options for configuring a **fill-graphic** with a repeating graphic:
+   
+   Using **external** to reference to an external graphic.
+
+   Use of **mark** to access a predefined shape. SLD provides several well-known shapes (circle, square, triangle, arrow, cross, star, and x). GeoServer provides additional shapes specifically for use as fill patterns.
+
+   Update `polygon_example` with the following built-in symbol as a repeating fill pattern:
+
+   .. code-block:: yaml
+      :emphasize-lines: 3-7
+      
+      symbolizers:
+      - polygon:
+          fill-graphic:
+            symbols:
+            - mark:
+                shape: square
+                fill-color: 'gray'
+
+#. The map preview (and legend) will show the result:
+   
+   .. image:: ../style/img/polygon_pattern_0.png
+   
+#. Add a black stroke:
+
+   .. code-block:: yaml
+      :emphasize-lines: 8,9
+
+      symbolizers:
+      - polygon:
+          fill-graphic:
+            symbols:
+            - mark:
+                shape: square
+                fill-color: 'gray'
+                stroke-color: 'black'
+                stroke-width: 1
```

Comment:
Adding it in for now, but this seems to be a regression occurring in a couple places with YSLD. 

@smithkm I expected SLD defaults to be respected (such as a black stroke) but that does not seem to be the case. Is that intensional?